dawn/src/dawn/Compiler/DawnCompiler.cpp

```diff
@@ -277,9 +275,7 @@ DawnCompiler::optimize(const std::map<std::string, std::shared_ptr<iir::StencilI
       optimizer.pushBackPass<PassTemporaryType>();
       optimizer.pushBackPass<PassLocalVarType>();
       optimizer.pushBackPass<PassRemoveScalars>();
-      // modify stages and their extents ...
-      optimizer.pushBackPass<PassComputeStageExtents>();
```

Comment:
should we make sure that this function is now called after `PassRemoveScalars` and `PassFixVersionedInputFields`?

Reply:
It is called in `PassSetSyncStage.run`. What type of check did you have in mind?